test/sql/utils/tsl_debugsupport.sql

```diff
@@ -0,0 +1,8 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+\ir testsupport.sql
+
+CREATE OR REPLACE FUNCTION ts_test_override_current_timestamptz(new_value TIMESTAMPTZ)
+RETURNS VOID AS :TSL_MODULE_PATHNAME, 'ts_test_override_current_timestamptz' LANGUAGE C VOLATILE STRICT;
```

Comment:
Since this file is referencing TSL files, shouldn't it be in something like `tsl/test/sql/utiils/debugsupport.sql`?

Reply:
```
ls -l ./tsl/test/sql/utils
lrwxrwxrwx 1 sven sven 23 Jun 22 16:19 ./tsl/test/sql/utils -> ../../../test/sql/utils
```